README.md

```diff
@@ -233,15 +239,25 @@ Replace example values with the appropriate values for your service to recommenc
 {
   "credId":"0000000026936462",
   "continueUrl":"/student-tax-checker/verify-email/journey-completed",
-  "origin":"stc",
+  "origin":"stc", // Will be displayed in the email as signature if labels.en.userFacingServiceName is null
   "deskproServiceName":"student-tax-checker",  // Optional, if absent then 'origin' will be used
   "accessibilityStatementUrl":"/accessibility-statement/stc",
-  "pageTitle": "Student Tax Checker", // Optional; will be displayed in the GOV.UK heading of the page
+  "pageTitle": "Student Tax Checker", // Optional, will be displayed in the email as sub title if labels.en.pageTitle is null
```

Comment:
"is null" => "is missing"

Reply:
The Lock Screen / English is now showing the Welsh version: 
https://github.com/hmrc/email-verification/blob/e9b04e43b52ce8758aa7589a1eb84d38b1d6d850/README.md#lock-screen---max-email-address-attempts-exceeded
Please revise the /doc-images/Locked_MaxEmailAttempts.png file, thx